ui/appui-react/src/appui-react/utils/Placement.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+type Side = "Left" | "Top" | "Right" | "Bottom";
+type Corner = "TopLeft" | "TopRight" | "BottomLeft" | "BottomRight" | "RightTop" | "LeftTop"
+
+/**
+ * Identifies the placement of a given element.
+ * @public
+ */
+export type Placement = Side | Corner;
```

Comment:
Let's define the `Alignment = "start" | "end";` and placement as ``` Placement = Side | `${Side}-${Alignment}` ```.

Reply:
fixed